service-worker.js

```diff
@@ -50,7 +68,7 @@ var regexpCachedContentTypes = /text\/css|text\/javascript|application\/javascri
  * 'example-extension' is included to show how to add another schema if necessary
  * @type {RegExp}
  */
-var regexpExcludedURLSchema = /^(?:chrome-extension|example-extension):/i;
+var regexpExcludedURLSchema = /^(?:file|chrome-extension|example-extension):/i;
```

Comment:
This disables use of Cache API when a particular schema is used. Currently listed are those that cause exceptions if an attempt is made to access Cache. Would we want a moz-extension schema that was fully supported by Firefox not to be able to use Cache API? In some ways it's immaterial, because the code will never currently run, and it's hare for us to test what would happen if Service Workers were supported in Firefox extensions!